### ISSUE TITLE
Error running app with dbus-launch as root

### ISSUE THREAD

**Issue by FarelRA** [skycocker/chromebrew#6987]
## Description

When I try to run the app with dbus-launch as root, I get an error
I've also tried this with various apps

As root I run:
```shell
dbus-launch gnome-terminal
```

I got this error:
```
# Zsh
dbus-launch: error while loading shared libraries: libSM.so.6: cannot open shared object file: No such file or directory

# Bash
Segmentation fault (core dumped)
```

## Reproducing

1. Press ```Ctrl + Alt + T```, and type ```shell```
2. Run an app with dbus-launch as root (e.g. ```sudo dbus-launch gnome-terminal```)
3. Or login as root and run an app (e.g. ```dbus-launch gnome-terminal```)

## Versions and system information
***
- Architecture: `x86_64` (`x86_64`)
- Kernel version: `5.10.106-brunch-sebanc`
- Chromebrew version: `1.23.12`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib64`
- Last update in local repository: `46400a1: util_linux -> 2.38 (#6983)`
- OS variant: `Chrome OS`
- OS version: `rammus-release/R100-14526.84.0`
- OS channel: `stable-channel`

CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=8414072
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={625849FA-56A0-4E67-9163-B89BE0C2A6AE}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={625849FA-56A0-4E67-9163-B89BE0C2A6AE}
CHROMEOS_RELEASE_BOARD=rammus-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=84
CHROMEOS_RELEASE_BUILDER_PATH=rammus-release/R100-14526.84.0
CHROMEOS_RELEASE_BUILD_NUMBER=14526
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=100
CHROMEOS_RELEASE_DESCRIPTION=14526.84.0 (Official Build) stable-channel rammus 
CHROMEOS_RELEASE_KEYSET=mp-v2
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=14526.84.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=14526.84.0
***

**Comment by satmandu** [comment]
You may have to export LD_LIBRARY_PATH in your zsh startup file in your
root profile so that it is used when running programs as root.

**Comment by FarelRA** [comment]
After I export ```LD_LIBRARY_PATH``` in zsh I get a different error message
```
[1]    26602 segmentation fault (core dumped)  dbus-launch gnome-terminal
```

**Comment by FarelRA** [comment]
Fixed by getting #6991 pull request changes locally and crew upgrade
